inst/tests/tests.Rraw

```diff
@@ -16400,6 +16400,12 @@ test(2124.1, format(as.ITime(t0)), '00:00:00')
 test(2124.2, format(as.IDate(t0)), '2019-10-01')
 Sys.setenv(TZ=oldtz)
 
+# Extra arguments in yaml
+data = data.table(x = 1, y = 100)
+file = tempfile()
+fwrite(data, file, yaml =  list(something_else = "o hai!"))
+x = fread(file, yaml = TRUE)
+test(2125, attr(x, "yaml_metadata")$something_else, "o hai!")
```

Comment:
another test that the rest of the `yaml_metadata` object is "as expected" would be nice, to be sure we're doing as advertised -- both writing the "standard" metadata & the user-supplied extras.

Just a test of a single key/value should be enough

Reply:
Something like `test(2125.02), attr(x, "yaml_metadata")$schema$fields[[1]]$name,  "x")`?

Reply:
yep that should work